src/sync.py

```diff
@@ -1,74 +1,91 @@
-import copy
+import time
 
-from src.jira import JiraIssue
-from src.zenhub import ZenHubIssue
+from src.jira import JiraRepo
+from src.zenhub import ZenHubRepo
 
 
 class Sync:
+    """Assumptions:
+    - Unito has run currently, so that
+        - Each issue has a counterpart in each management system
```

Comment:
Good thinking. Let's discuss whether this is the best solution we can find. Alternatively, we could test whether such counterpart exists, and if not do something about it. That's what I have been implementing in #49 as part of the spint class, but not I'm wondering whether that is the best design. Let's discuss.